cfgov/jinja2/v1/landing-page/index.html

```diff
@@ -136,4 +136,13 @@ <h2 class="o-half-width-link-blob-group__header">
             {{ render_stream_child(block) }}
         {% endif %}
     {% endfor %}
+
+    {% if page.email_signup or prototype %}
+        {% if prototype %}
```

Comment:
This should really be wrapping the entire content in the block. In other words, the block should start with 
```
{% block some_content %}
    {% if prototype %}
        {{ some prototype stuff }} 
    {% else %} 
        {{ actual django/wagtail data }}
    {% endif %} 
{% endblock %}
```